main.go

```diff
@@ -85,6 +85,11 @@ func main() {
 		setupLog.Error(err, "unable to start manager")
 		os.Exit(1)
 	}
+	setupLog.Info("This is what logs look like by default")
```

Comment:
What is the purpose of this line since the logger is set up before and `setupControllerConfig` is not going to change anything? Or it was used locally and it should be removed now?

![Screenshot_20201229_114437](https://user-images.githubusercontent.com/5887312/103275294-8b0d6900-49cc-11eb-9ff6-e1a59bf49986.png)

Reply:
It's a leftover from testing that I forgot to remove -- sorry (was testing to see if we can update the logger to add workspace_id after its created -- we can't)

Reply:
Fixed.